packages/runtime/datastore/src/localChannelContext.ts

```diff
@@ -4,6 +4,7 @@
  */
 
 import { strict as assert } from "assert";
+import { cloneDeep as clone } from "lodash";
```

Comment:
any reason not to have simple import { cloneDeep } ?

Reply:
Changed it